back_office/products/migrations/0005_alter_product_price.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.7 on 2024-01-06 16:16
```

Comment:
At least for this commit can you delete these migrations files as well?

Reply:
deleted migration files 0005 and 0006